Tools/Scripts/webkitdirs.pm

```diff
@@ -3011,7 +3012,7 @@ Options specific to macOS:
   --locale=LOCALE                   Use a specific locale instead of the system region.
 
 Options specific to iOS:
-  --iphone-simulator                Run the app in an iPhone Simulator
+  --ios-simulator                   Run the app in an iPhone Simulator
   --ipad-simulator                  Run the app in an iPad Simulator
```

Comment:
Someone will need to check the actual behavior as part of reviewing, but it would be very incorrect if we opposed "ios" and "ipad" here. Other tools either have "iphone" and "ipad", or only "ios". 

Given that this tool needs to launch a simulator, differentiating between these makes sense, so I think that the usage text describes how it should probably behave in reality, even if it does not.